src/lib/messenger.js

```diff
@@ -23,6 +23,19 @@ export default function messenger(bot) {
       const text = message.text.toLowerCase();
       const user = message.user || {};
 
+      function stripText(input) {
+        let commandText = '';
+        let command = '';
+
+        if (input.indexOf(' ') !== -1 && input.startsWith('/')) {
```

Comment:
this logic would be easier to comprehend if it is extracted into its own function with a descriptive function name

and a side question: we only want to activate a command when it's occurring first in a text? not in the middle or end of a text?